src/sframe/csv_line_tokenizer.cpp

```diff
@@ -199,6 +214,33 @@ size_t csv_line_tokenizer::tokenize_line(char* str, size_t len,
                        --ctr;
                      });
 
+  if (!success || ctr < output.size()) {
+    std::stringstream strm;
+    if (!tokenizer_impl_error.empty()) strm << tokenizer_impl_error << "\n";
+    if (tokenizer_impl_fail_pos >= 0 && (size_t)tokenizer_impl_fail_pos <= len) {
+      strm << "Parse failed at token ending at: \n";
+      std::string line(str, len);
+      // colorize the line.
+      line.insert(tokenizer_impl_fail_pos, "\033[1m\033[31m^\033[00m");
+      if (line.length() > 256) {
+        // truncate around the fail pos
+        ssize_t start = std::max<ssize_t>(0, tokenizer_impl_fail_pos - 60);
+        ssize_t end = std::min<ssize_t>(len, tokenizer_impl_fail_pos + 60);
+        line = line.substr(start, end - start);
```

Comment:
Do we need to turn off the colorization?  (Also, I'm worried about error reporting in XCode; the log there doesn't support colorization.)

Reply:
what do you recommend here?

Reply:
Is there a flag I can check to optionally turn off colorization?

Reply:
There seems to be definition in logger.hpp `#define COLOROUTPUT`. But it seems like it is forced to be set. Are you using this?